docs/tutorials/build_a_node_in_rust.md

```diff
@@ -693,28 +693,37 @@ fn handle_ldk_event(..) {
 
 **References:** [`Event` docs](https://docs.rs/lightning/*/lightning/util/events/enum.Event.html), [LDK sample node event handling example](https://github.com/lightningdevkit/ldk-sample/blob/bc07db6ca4a3323d8718a27f85182b8157a20750/src/main.rs#L101-L240), [`EventHandler` docs](https://docs.rs/lightning/*/lightning/util/events/trait.EventHandler.html)
 
-### 17. Initialize the `ChannelManagerPersister`
+### 17. Initialize the `Persister`
 
 **What it's used for:** keeping `ChannelManager`'s stored state up-to-date
 
 **Example:**
 
 ```rust
-// In this example, we provide a closure to handle ChannelManager persistence.
-// But you're also free to provide an implementation of the trait
-// `ChannelManagerPersister` instead.
+// In this example, we provide an implementation of the trait `Persister`
+
+struct YourDataPersister()
```

Comment:
Will need to make this struct definition match the one used in the `let persister = DataPersister { data_dir: ldk_data_dir.clone() };` line below.